core/src/test/java/org/ehcache/events/CacheEventNotificationServiceImplTest.java

```diff
@@ -64,7 +65,8 @@ public void setUp() {
     orderedExecutor = Executors.newSingleThreadExecutor();
     unorderedExecutor = Executors.newCachedThreadPool();
     store = mock(Store.class);
-    eventService = new CacheEventNotificationServiceImpl<Number, String>(orderedExecutor, unorderedExecutor, store);
+    TestTimeSource timeSource = new TestTimeSource();
```

Comment:
Same question as with the previous `TestTimeSource` inclusion - which reinforces the point about moving `SystemTimeSource`